CameraID: 1
<<<<<<< HEAD
Hmax: 70
Hmin: 0
Smax: 255
Smin: 180
Vmax: 255
Vmin: 80
Width: 480
Height: 360

=======
Bmax: 70
Bmin: 0
Gmax: 255
Gmin: 180
Rmax: 255
Rmin: 130
Width_image: 480
Height_image: 360
Limit_close_distance: 100
Limit_mid_distance: 500
 
>>>>>>> 32ffeead7f782714fc715b0f4ce9b66228786980
